setup.py

```diff
@@ -17,7 +17,7 @@
 import setuptools
 
 
-DEPENDENCIES = []
+DEPENDENCIES = ['oauth2client', 'pyu2f']
```

Comment:
`oauth2client` is covered by the `oauth2client` extra, e.g., you would install this using `pip install google-reauth[oauth2client]`.

Reply:
still need to address this.

Reply:
If I remove it from dependencies, nox seems to ignore EXTRAS, I can't figure out why.

Reply:
Get nox to run `install .[oauth2client]` and you should be fine.